src/test/java/org/generationcp/breeding/manager/listmanager/listcomponent/ListViewActionMenuTest.java

```diff
@@ -135,7 +135,8 @@ public void testLayoutAdminLink() {
 
 	  	final ContextMenu.ContextMenuItem codingAndGroupingOptions = Mockito.mock(ContextMenu.ContextMenuItem.class);
 		this.menu.setCodingAndGroupingOptions(codingAndGroupingOptions);
-		this.menu.layoutAdminLink();
+		this.menu.addDeleteGermplasmLink();
```

Comment:
should we call this function here? I mean if the test is refering to Ungroup only?  
Conceptually this test should be also splitted in 2 but if we are testing both in one then we need to rename the funtion to something more generic